.github/actions/create-release-candidate-v1/action.yml

```diff
@@ -0,0 +1,224 @@
+      shell: bash
+      run: |
+        # https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#example-of-a-multiline-string
+        {
+          echo 'RELEASE_NOTES<<EOF'
+          git show \
+            --no-color \
+            --no-prefix \
+            --output-indicator-new=! CHANGELOG.md | egrep '^!' | awk -F'^[!]' '{print $2}' | sed -e 's/\n/$0A/g'
+          echo EOF
+        } >> $GITHUB_ENV
+
+    # https://github.com/dequelabs/axe-core-npm/blob/62fd241fb3748ca8ce4adb30d7976947ccaa18ee/.github/workflows/create-release.yml#L32-L57
+    - name: Create pull request
+      if: ${{ !steps.is_release_in_progress.outputs.in_progress }} && ${{ steps.has_releasable_commits.outputs.should_release }}
+      # Last commit on v5 tag: https://github.com/peter-evans/create-pull-request/commit/153407881ec5c347639a548ade7d8ad1d6740e38
+      uses: peter-evans/create-pull-request@153407881ec5c347639a548ade7d8ad1d6740e38
+      id: create_pull_request
+      with:
+        token: ${{ inputs.token }}
+        branch: release-${{ steps.get_release_date.outputs.release-date }}
+        base: release
+        title: 'chore: release v${{ steps.prepare_release.outputs.version }}'
+        body: ${{ env.RELEASE_NOTES }}
+        # If there are any changes not already committed, they will be added to
+        # a commit with this as the message.
+        # If there are no changes no commit will be created.
+        commit-message: 'chore: applying release changes'
+
+    - name: Create release notes issue
+      if: ${{inputs.docs-repo != 'null' }} && ${{ !steps.is_release_in_progress.outputs.in_progress }} && ${{ steps.has_releasable_commits.outputs.should_release }}
+      uses: dequelabs/axe-api-team-public/.github/actions/create-project-issue-v1@main
+      id: create_release_notes_issue
+      with:
+        token: ${{ inputs.token }}
+        repo: ${{ inputs.docs-repo }}
+        labels: 'PRIORITY: high,release'
+        assignees: 'enlarsen'
+        column_name: 'To Do'
+        title: '${{ github.repository }} v${{ steps.prepare_release.outputs.version }} - Release Notes Needed'
+        body: |
+          PR: ${{ steps.create_pull_request.outputs.pull-request-url }}
+
+          Release Notes are needed for ${{ github.repository }} v${{ steps.prepare_release.outputs.version }}. Please make sure this issue is closed when the PR is merged.
+
+    - name: Create release issue
+      if: ${{ !steps.is_release_in_progress.outputs.in_progress }} && ${{ steps.has_releasable_commits.outputs.should_release }}
+      uses: dequelabs/axe-api-team-public/.github/actions/create-project-issue-v1@main
+      id: create_release_issue
+      with:
+        token: ${{ inputs.token }}
+        labels: 'PRIORITY: high,release'
+        column_name: 'Ready for QA'
+        title: '${{ github.repository }} v${{ steps.prepare_release.outputs.version }}'
+        body: |
+          Below is everything related to this release 
```

Comment:
Is there anyway we could move this to a local file of this action that it can read from rather than hardcoding it into the action step itself?

Reply:
Unfortunately not as we include outputs in specific spots in that template